bigbluebutton-html5/imports/api/meetings/server/modifiers/addMeeting.js

```diff
@@ -64,6 +91,7 @@ export default function addMeeting(meeting) {
       screenshareConf: String,
     },
     metadataProp: Object,
+    lockSettingsProps: Match.Any,
```

Comment:
` Match.ObjectIncluding(`...

Reply:
This value can be null, can we use this option in this case? I tried Match.Maybe(... but could not handle that to work.